.github/workflows/static.yaml

```diff
@@ -8,7 +8,7 @@ on:
 
 jobs:
   static_checks:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
This is not needed. I will revert it. Note that we might want to use a pinned release, so that we decide when to upgrade and don't let Github to decide for ourselves :)